MastodonSDK/Sources/MastodonUI/Helper/MediaLayoutHelper.swift

```diff
@@ -0,0 +1,314 @@
+//
+//  MediaLayoutHelper.swift
+//  
+//
+//  Created by Grishka on 25.03.2023.
+//
+
+import Foundation
+import MastodonSDK
+import CoreDataStack
+
+public struct MediaLayoutResult {
+	let width: Int
+	let height: Int
+	let columnSizes: [Int]
+	let rowSizes: [Int]
+	let tiles: [Tile]
+	
+	public struct Tile {
+		var colSpan: Int
+		let rowSpan: Int
+		var startCol: Int
+		let startRow: Int
+		var width: Int = 0
+	}
+}
+
+class MediaLayoutHelper {
+	static let maxWidth: Float = 1000
+	static let maxHeight: Float = 1777
+	static let minHeight: Float = 563
+	static let gap: Float = 1.5
+	static let maxRatio: Float = Float(maxWidth) / Float(maxHeight)
```

Comment:
In this case you don’t need any type annotations or explicit casts:
```suggestion
	static let maxRatio = maxWidth / maxHeight
```